alerts.md

```diff
@@ -1,40 +1,39 @@
-Alerts
-======
+# Alerts
```

Comment:
According to the style guide for ZD friendliness [1], this should be underlined with `=====` instead of starting with `#`. I just learned about this style guide today!

[1] https://github.com/kubecost/docs/blob/main/markdown.md#titles

Reply:
Oh nice! I hadn't seen that either.

The reformatting was Prettier defaults. I promise I wasn't just going around changing `===` to `#` and `*` to `-`